include/swift/Option/FrontendOptions.td

```diff
@@ -357,6 +357,9 @@ def import_module : Separate<["-"], "import-module">,
 def print_stats : Flag<["-"], "print-stats">,
   HelpText<"Print various statistics">;
 
+def debugger_testing_transform : Flag<["-"], "debugger-testing-transform">,
+  HelpText<"Apply a transform to aid testing of the debugger">;
```

Comment:
Perhaps: "Instrument the code with calls to an intrinsic that record the expected values of local variables so they can be compared against the results from the debugger."

Reply:
Thanks, I'll use this.